pkg/apis/v1/ec2nodeclass_hash_test.go

```diff
@@ -184,21 +184,18 @@ var _ = Describe("Hash", func() {
 		hash := nodeClass.Hash()
 
 		// Update a behavior/dynamic field
-		nodeClass.Spec.SubnetSelectorTerms = []v1.SubnetSelectorTerm{
-			{
-				Tags: map[string]string{"subnet-test-key": "subnet-test-value"},
-			},
-		}
-		nodeClass.Spec.SecurityGroupSelectorTerms = []v1.SecurityGroupSelectorTerm{
-			{
-				Tags: map[string]string{"sg-test-key": "sg-test-value"},
-			},
-		}
-		nodeClass.Spec.AMISelectorTerms = []v1.AMISelectorTerm{
-			{
-				Tags: map[string]string{"ami-test-key": "ami-test-value"},
-			},
-		}
+		nodeClass.Spec.SubnetSelectorTerms = []v1.SubnetSelectorTerm{{
+			Tags: map[string]string{"subnet-test-key": "subnet-test-value"},
+		}}
+		nodeClass.Spec.SecurityGroupSelectorTerms = []v1.SecurityGroupSelectorTerm{{
+			Tags: map[string]string{"sg-test-key": "sg-test-value"},
+		}}
+		nodeClass.Spec.AMISelectorTerms = []v1.AMISelectorTerm{{
+			Tags: map[string]string{"ami-test-key": "ami-test-value"},
+		}}
+		nodeClass.Spec.SubnetSelectorTerms = []v1.SubnetSelectorTerm{{
```

Comment:
```suggestion
		nodeClass.Spec.CapacityReservationSelectorTerms = []v1.SubnetSelectorTerm{{
```

Reply:
I'm assuming that this doesn't fail the test because these terms aren't part of the drift hash calculation?

Reply:
Yeah, definitely still going to fix this though